commit 60f85f11594497f9b16518fd105b84cde8ad9f9d (refs/changes/37/1437/1)
Author: YangYong3 <yy1111.yang@samsung.com>
Date:   2017-02-06T11:20:18+09:00 (2 years, 8 months ago)
    
    joyent/node-triton#170 Integration tests fail to change company name back to original value ...

diff --git a/test/integration/cli-account.test.js b/test/integration/cli-account.test.js
index 04b42b1..aef464e 100644
--- a/test/integration/cli-account.test.js
+++ b/test/integration/cli-account.test.js
@@ -100,7 +100,8 @@ test('triton account', function (tt) {
 
     tt.test(' triton account update companyName=<oldvalue>', writeTestOpts,
             function (t) {
-        h.triton('account update companyName=' + account.companyName || '',
+        var CPNAME = '\'' + account.companyName + '\'' || '';
+        h.triton('account update companyName=' + CPNAME,
                 function (err, _o, _e) {
             if (h.ifErr(t, err))
                 return t.end();
